Neos.Fusion/Classes/ViewHelpers/FusionContextTrait.php

```diff
@@ -33,18 +33,19 @@ trait FusionContextTrait
      */
     protected function getContextVariable($variableName)
     {
-        $value = null;
-
         $view = $this->viewHelperVariableContainer->getView();
         if ($view instanceof FusionAwareViewInterface) {
             $fusionObject = $view->getFusionObject();
             $currentContext = $fusionObject->getRuntime()->getCurrentContext();
             if (isset($currentContext[$variableName])) {
-                $value = $currentContext[$variableName];
+                return $currentContext[$variableName];
+            }
+            $fusionGlobals = $fusionObject->getRuntime()->fusionGlobals;
+            if ($fusionGlobals->has($variableName)) {
+                return $fusionGlobals->get($variableName);
```

Comment:
Just wondering:
Is this order of evaluation (context first, then global context) something that is always the same.. And didn't we even consider adding a dedicated method to the runtime in order to centralize this fallback behavior?

Reply:
see https://github.com/neos/neos-development-collection/blob/9.0/Neos.Fusion/Classes/Core/Runtime.php#L751C1-L751C1

Reply:
We decided against adding a method to the Runtime were one can access both types of context via one api. Only for eel was this custom implemented.

Reply:
@bwaidelich we will merge this once you consider this solved